website/docs/d/canonical_user_id.html.markdown

```diff
@@ -10,7 +10,9 @@ description: |-
 # Data Source: aws_canonical_user_id
 
 The Canonical User ID data source allows access to the [canonical user ID](http://docs.aws.amazon.com/general/latest/gr/acct-identifiers.html)
-for the effective account in which Terraform is working.
+for the effective account in which Terraform is working.  
+
+**NOTE:** [This value](https://docs.aws.amazon.com/AmazonS3/latest/API/RESTServiceGET.html) is only included in the response in the US East (N. Virginia), US West (N. California), US West (Oregon), Asia Pacific (Singapore), Asia Pacific (Sydney), Asia Pacific (Tokyo), EU (Ireland), and South America (São Paulo) regions.
```

Comment:
FYI the canonical owner ID is still returned in other regions, its just the display name that is missing.

Reply:
Updated in PR #6792 .